codice/ddf
# DDF-4383 Make Preview, History, Associations, Quality, Archive, and Overwrite Inspector tabs in Intrigue optional.

#### What does this PR do?
Make "Preview" tab in the Inspector optional. Hide "History", "Associations", "Quality", "Archive", "Overwrite" tabs in the Inspector when Allow Editing is disabled
#### Who is reviewing it? 
@Bdthomson @hayleynorton @alanchampion 
#### Select relevant component teams: 
@codice/docs 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@bdeining 
#### How should this be tested?
- build/run 
- ingest data in a list
- open the inspector for a result and observe that all tabs are present
- in Admin > Search UI > Catalog UI Search config uncheck Allow Editing
- verify that the History, Associations, Quality, Archive, and Overwrite Inspector tabs are not visible
#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4383](https://codice.atlassian.net/browse/DDF-4383)
#### Screenshots
<img width="915" alt="screen shot 2018-12-11 at 3 28 48 pm" src="https://user-images.githubusercontent.com/39923178/49834408-8b8d2580-fd59-11e8-8ad3-13b98d2da42f.png">


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
